code/datums/emergency_calls/deliverance.dm

```diff
@@ -0,0 +1,69 @@
+/datum/emergency_call/deliverance
+	name = "Unitologist"
+	pref_name = "Unitologist Missionary"
+	weigh = 25
+	landmark_tag = "unitologiststeam"
+
+/datum/emergency_call/deliverance/print_backstory(mob/living/carbon/human/H)
+	to_chat(H, "<B>You are Unitologist.</b>")
+	to_chat(H, "")
+	to_chat(H, "<B>Follow any orders directly from Unitologist Central Command.</b>")
+
+/datum/emergency_call/deliverance/create_member(datum/mind/M)
```

Comment:
why is all of this stuff in the deliverance subtype? Most of this seems like generic behaviour. Mind transfer, outfit picking, skills, namechanging, etc